bin/sirius-script-create-mamba-env.bash

```diff
@@ -0,0 +1,558 @@
+#!/usr/bin/env bash
+
+# Exit if any command fails
+# set -e
+
+##############################################################################
+# Parse arguments.
+# Adapted from: https://stackabuse.com/how-to-parse-command-line-arguments-in-bash/
+
+function help
+{
+    echo "Usage: sirius-script-create-mamba-env.bash
+    [ -n | --no-clone-repos ] Instead of cloning, find repos in system
+    [ --no-sim ] Do not install simulation packages.
+    [ --no-ioc ] Do not install IOC related packages.
+    [ --no-ima ] Do not install magnets simulation packages.
+    [ --no-colleff ] Do not install collective effects packages.
+    [ --root-lnls-fac ] Root folder for lnls-fac repos. Defaults to /.
```

Comment:
@fernandohds564 , did you mean `./` instead of `/.` in these lines?

Reply:
No, the folder is the root folder and the dot is just a marker for the end of the sentence.

Reply:
Do you think I should change the message? How could I avoid this quasi-degenerescence?

Reply:
`Defaults to "/".`

Reply:
done!